app/jobs/expire_token_job.rb

```diff
@@ -0,0 +1,9 @@
+class ExpireTokenJob
+  include Sidekiq::Worker
+
+  def perform(params = {})
```

Comment:
Use explicit arguments, or at least key arguments (:+1:) here. 

We haven't defined constructor for job to provide attributes assigning, call them from the `params` hash all the time looks messy.